bot/cogs/dogeboard.py

```diff
@@ -0,0 +1,284 @@
+            else:
+                embed.add_field(name="Emoji", value=dogeboard_data.dogeboard_emoji)
+                embed.add_field(
+                    name="Channel",
+                    value=ctx.guild.get_channel(dogeboard_data.dogeboard_id).mention
+                )
+                embed.add_field(name="Required", value=str(dogeboard_data.dogeboard_reactions_required))
+
+            await ctx.send(embed=embed)
+
+    @dogeboard.command(brief="Set the dogeboard_emoji used for the DogeBoard, Custom Emojis Only")
+    async def emoji(self, ctx: Context) -> None:
+        """Set the dogeboard_emoji used for the DogeBoard."""
+        dogeboard_data = await self._get_guild_data(ctx.guild.id)
+
+        if not dogeboard_data:
+            await ctx.send("> DogeBoard has not been configured for this guild. "
+                           "Use `.dogeboard channel <channel>` to enable")
+            return
+
+        reaction_msg = await ctx.send(content="> React with the new dogeboard_emoji")
+
+        def check(reaction: Reaction, member: Member) -> bool:
+            return member == ctx.author and reaction_msg == reaction.message
+
+        try:
+            reaction, member = await self.bot.wait_for("reaction_add", check=check, timeout=60)
+        except asyncio.TimeoutError:
+            return await reaction_msg.edit(content="> You didn't react, Canceling dogeboard_emoji change.")
+        finally:
+            await reaction_msg.delete(delay=10)
+
+        # Default Emoji or Guild dogeboard_emoji
+        if not reaction.custom_emoji or reaction.emoji in ctx.guild.emojis:
+            dogeboard_data.dogeboard_emoji = reaction.emoji
+        else:
+            await ctx.send("> Please react with a default or guild dogeboard_emoji.")
+            return None
+
+        await ctx.send(f"> Updated your emoji to {dogeboard_data.dogeboard_emoji}")
+        await self._update_guild_data(dogeboard_data)
+
+    @dogeboard.command(brief="Set the channel for DogeBoard messages to be sent to.")
+    async def channel(self, ctx: Context, channel: TextChannel) -> None:
+        """Set the channel used for the DogeBoard."""
+        dogeboard_data = await self._get_guild_data(ctx.guild.id)
+
+        if not dogeboard_data:
+            dogeboard_data = DogeBoardData(guild_id=ctx.guild.id, channel_id=channel.id)
+        else:
+            dogeboard_data.dogeboard_id = channel.id
+
+        await self._update_guild_data(dogeboard_data)
+        await ctx.send(f"> Updated the channel for DogeBoard to: {channel.mention}", delete_after=5)
+
+    @dogeboard.command(brief="Set the reactions required for the DogeBoard")
+    async def required(self, ctx: Context, amount: int) -> None:
+        """Set the amount of required emojis to trigger the DogeBoard."""
+        dogeboard_data = await self._get_guild_data(ctx.guild.id)
+
+        if not dogeboard_data:
+            await ctx.send("> DogeBoard has not been configured for this guild. "
```

Comment:
There should be clearer instructions as to what can be configured. It should mention the ability to change the reaction and set the reactions required.